src/commands/metafrage.ts

```diff
@@ -0,0 +1,79 @@
+"use strict";
+
+// ========================= //
+// = Copyright (c) NullDev = //
+// ========================= //
+
+// Dependencies
+let parseOptions = require("minimist");
+
+/**
+ * Sends instructions on how to ask better questions
+ * @param {import("discord.js").Client} client
+ * @param {import("discord.js").Message} message
+ * @param {Array} args
+ * @param {Function} callback
+ * @returns {Function} callback
+ */
+exports.run = (client: any, message: any, args: any, callback: any) => {
+    // parse options
+    let options = parseOptions(args, {
+        "boolean": [
+            "english"
+        ],
+        alias: {
+            english: "e"
+        }
+    });
+
+    if (args.length === 0 || (args.length === 1 && options.english)){
+        // insult collections, feel free to expand
+        let germanInsults: string[] = [
+            "du Sohn einer ranzigen Hafendirne!",
+            "möge dich der Blitz beim scheißen treffen!",
+            "du verdammter Troglodyt!",
+            "sonst muss ich heute Nacht noch deine Mama besuchen!"
+        ];
+        let englishInsults: string[] = [
+            "you fucking imbecile!",
+            "retard!",
+            "you troglodyte!"
+        ];
+        // by default, just pick the first or a fixed insult
+        let choice: number = 0;
+        let insult: string = "Idiot!";
+
+        if (!options.english){
+            // insult user, then explain meta questions
+            if (germanInsults.length !== undefined){
+                choice = Math.max(choice, Math.floor(Math.random() * germanInsults.length));
+                insult = germanInsults[choice];
+            }
+            message.channel.send(`
+            Hör auf, Metafragen zu stellen, ${insult}
+            Das ist reine Zeitverschwendung und hindert uns nur daran, ~~uns zu beleidigen~~ an echten Problemen zu arbeiten.
+            Für Tipps zum besser machen: <https://metafrage.de>
+            `);
+        }
+        else{
+            // insult user, then explain meta questions - but this time in english!
+            if (englishInsults.length !== undefined){
```

Comment:
Hö? Was soll das denn?

Reply:
Gedanke war, dass wenn array leer -> Verwende die Default-Insult aus L44. Ansonsten wird zufällig eine aus dem Array ausgewählt. Wenn das deines Erachtens nicht so sinnvoll ist, kann ich das gerne anpassen.

Reply:
Es geht eher um die Art des Checks, weil length nicht `undefined` sein kann, weil wegen Prototypeneigenschaft des Arrays. Also wenn du den Check machen willst (auch wenn wir ja bei der Definition sehen, dass das Array nicht leer ist), würde ich eher so checken:

```suggestion
            if (englishInsults.length > 0){
```

Reply:
Nochmal als Ergänzung dazu:

Ich finde diesen Check ziemlich unnötig, weil wir **wissen** dass das Array nicht leer ist. Weil dann müsstest du so konsistent sein und **alles** auf undefined checken, obwohl man mit static code analysis fehler entdecken würde. 

If that makes any sense...

Reply:
Ergibt Sinn, ja. Der Code war tatsächlich vor dem array da, hatte das zuerst als Liste in einer Datei überlegt. Aber stimmt, als static code kann ich den Check auch sein lassen.

Reply:
Hab den Check jetzt entfernt, da nicht notwendig.